aim/sdk/plotly_sequence.py

```diff
@@ -0,0 +1,38 @@
+from typing import Union, Tuple
+
+from aim.sdk.sequence import Sequence
+from aim.sdk.objects.plotly import Plotly
+
+
+class Figures(Sequence):
+    """Class representing series of Plotly figure objects or Plotly lists."""
+
+    @classmethod
+    def allowed_dtypes(cls) -> Union[str, Tuple[str, ...]]:
+        plotly_typename = Plotly.get_typename()
+        return plotly_typename, f'list({plotly_typename})'
+
+    @classmethod
+    def sequence_name(cls) -> str:
+        return 'figures'
+
+    def first_step(self):
+        """Get sequence tracked first step.
+
+        Required to implement ranged and sliced data fetching.
+        """
+        return self._meta_tree['first_step']
+
+    def last_step(self):
+        """Get sequence tracked last step.
+
+        Required to implement ranged and sliced data fetching.
+        """
+        return self._meta_tree['last_step']
+
+    def record_length(self):
```

Comment:
we don't need this method since only single figures are allowed to track

Reply:
removed with bunch of refactoring that it has caused